MultipleImages/index.jsx

```diff
@@ -0,0 +1,62 @@
+import React, { PropTypes } from 'react';
+import uuid from 'uuid';
+import styles from './style.css';
+import Image from '../Image';
+
+const getUrlAmount = (urls) => {
+  if (urls.length <= 2) {
```

Comment:
Heya @stevemdixon! 😄 
Curious if we might rename this function to something that reflects what it's doing a little better? Something like `getImageWrapperStyle` or `getImageWrapperHeight` since it looks like it returns style. Curious to hear your thoughts 😃